tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -950,11 +950,11 @@ public:
    RResultPtr<::TH1D> Histo1D(std::string_view vName, std::string_view wName)
    {
       // We build name and title based on the value and weight column names
-      auto h_name_title = std::string(vName);
-      h_name_title += "_";
-      h_name_title += wName;
-      auto h_nt_cstr = h_name_title.c_str();
-      return Histo1D<V, W>({h_nt_cstr, h_nt_cstr, 128u, 0., 0.}, vName, wName);
+      const auto vName_str = std::string(vName);
+      const auto wName_str = std::string(wName);
+      const auto h_name = vName_str + "_" + wName_str;
+      const auto h_title = h_name + ";" + vName_str + ";";
```

Comment:
maybe `vName*wName` would be clearer than `vName_wName`?

also, not sure: wouldn't it be more correct to have the weight mentioned in the x axis title?

Reply:
nice comment. Implemented.